project-base/app/phpunit.xml

```diff
@@ -42,7 +42,9 @@
         </testsuite>
         <testsuite name="Functional">
             <directory>tests/App/Functional</directory>
-            <directory>tests/*Bundle/Functional</directory>
+        </testsuite>
+        <testsuite name="FrontendApiFunctional">
+            <directory>tests/FrontendApiBundle/Functional</directory>
```

Comment:
Shouldn't this be mentioned in the upgrade notes?

Reply:
I'm not sure how to approach the upgrade notes in this version.  
I thought it will be in some form of general upgrade notes for the whole version. For me, it seems unhelpful to have info like 
"Make all the changes we do in the project base and btw. split this suite in two"

Reply:
added upgrade notes